timeless/reservations/views.py

```diff
@@ -7,14 +7,16 @@
 
 from timeless import DB
 from timeless.reservations.forms import ReservationForm
-from timeless.restaurants.models import Reservation
+from timeless.restaurants.models import Reservation, ReservationStatus
 from timeless import views
 from timeless.access_control.views import SecuredView
 from timeless.reservations import models
 
+from flask_sqlalchemy import SQLAlchemy
 
-bp = Blueprint("reservations", __name__, url_prefix="/reservations")
 
+bp = Blueprint("reservations", __name__, url_prefix="/reservations")
+db = SQLAlchemy()
```

Comment:
@marceloamadeu constant names should be all upper case (change `db` to `DB`) and maybe `bp` too if you have time for it.

Reply:
@ammaratef45 Adjusted. No need because DB already has `import`. I tried changing bp to BP and generated other errors. I also noticed that in all views there is no big DB. We can check with the @emilianodellacasa  if we create a headache to correct this.